packages/blocks/src/surface-block/surface-block.ts

```diff
@@ -231,6 +240,38 @@ export class SurfaceBlockComponent extends BlockElement<SurfaceBlockModel> {
     );
   }
 
+  private _updateViewportEvents() {
```

Comment:
Should be `initViewportEffects`. The sementic of `update` is to be explicitly called by some caller, rather than subscribing events.